package.json

```diff
@@ -1,15 +1,70 @@
 {
-  "name": "cmr-stac",
-  "version": "1.0.0",
-  "description": "An implementation of the STAC API on top of NASA's [Common Metadata Repository](https://cmr.earthdata.nasa.gov/search/).",
+  "name": "CMR-STAC",
+  "version": "2.0.0",
+  "description": "NASA' Common Metadata Repository STAC implementation.",
+  "repository": "https://github.com/NASA/cmr-stac",
+  "license": "NASA-1.3",
   "scripts": {
-    "postinstall": "node bin/install.js;",
-    "test": "echo 'Testing All Packages:'",
-    "posttest": "node bin/test.js",
-    "start": "cd search; npm start"
+    "test": "cross-env IS_LOCAL=true mocha",
+    "test:coverage": "cross-env IS_LOCAL=true nyc --reporter=html --reporter=text mocha",
+    "test:watch": "cross-env IS_LOCAL=true mocha --watch --reporter=dot",
+    "dev": "serverless offline start --reloadHandler --config serverless.dev.yml --stage dev",
+    "deploy": "sls deploy",
+    "remove": "sls remove"
   },
-  "repository": "https://github.com/nasa/cmr-stac",
-  "author": "Element 84",
-  "license": "Apache-2.0",
-  "dependencies": {}
+  "devDependencies": {
+    "@serverless/event-mocks": "^1.1.1",
+    "@serverless/typescript": "^3.21.0",
+    "@types/aws-lambda": "^8.10.108",
+    "@types/chai": "^4.3.4",
+    "@types/compression": "^1.7.2",
+    "@types/cookie-parser": "^1.4.3",
+    "@types/cors": "^2.8.12",
+    "@types/express": "^4.17.14",
+    "@types/mocha": "^10.0.0",
+    "@types/morgan": "^1.9.3",
+    "@types/node": "^18.11.9",
+    "@types/serverless": "^3.12.8",
+    "@types/sinon": "^10.0.13",
+    "@types/sinon-chai": "^3.2.9",
+    "@types/supertest": "^2.0.12",
+    "@types/whatwg-url": "^11.0.0",
+    "ajv": "^8.11.0",
+    "ajv-formats-draft2019": "^1.6.1",
+    "chai": "^4.3.7",
+    "cross-env": "^7.0.3",
+    "json-schema-to-typescript": "^11.0.2",
+    "mocha": "^10.1.0",
+    "mocha-suppress-logs": "^0.3.1",
+    "nyc": "^15.1.0",
+    "serverless": "^3.24.1",
```

Comment:
We've run into problems here with serverless (and _all_ plugins) needing to be dependencies, not devDependencies. Deployments fail without those packages. Though, this is probably just because when we deploy we use `npm ci --production` to skip over installing the dev packages

Reply:
We've been successfully deploying with the current setup and don't have an excessively large package being uploaded. Will verify